lib/routes/account.js

```diff
@@ -581,6 +591,16 @@ module.exports = function (
             doSigninConfirmation = mustVerifySession && emailRecord.emailVerified
           }
 
+          let flowCompleteSignal
+          if (service === 'sync') {
+            flowCompleteSignal = 'account.signed'
+          } else if (doSigninConfirmation) {
+            flowCompleteSignal = 'account.confirmed'
+          } else {
+            flowCompleteSignal = 'account.login'
+          }
```

Comment:
These conditional blocks being duplicated worries me that it'd be easy to forget to change one...